ziti/edge/conn.go

```diff
@@ -131,8 +131,10 @@ func (ec *MsgChannel) WriteTraced(data []byte, msgUUID []byte, hdrs map[int32][]
 		msg.Headers[UUIDHeader] = msgUUID
 	}
 
-	for k, v := range hdrs {
-		msg.Headers[k] = v
+	if hdrs != nil {
```

Comment:
is this needed? range over nil map is safe -- skips the loop

Reply:
Yeah, I could have sworn I hit a panic on this, but you are correct, it should not panic. If I see the panic again, I'll dig in deeper.